docs/source/markdown/podman-create.1.md

```diff
@@ -672,11 +678,16 @@ Set the network mode for the container. Invalid if using **--dns**, **--dns-opt*
 
 Valid _mode_ values are:
 
-- **bridge**: Create a network stack on the default bridge. This is the default for rootfull containers.
+- **bridge[:OPTIONS,...]**: Create a network stack on the default bridge. This is the default for rootfull containers. It is possible to specify these additional options:
+  - **alias=name**: Add network-scoped alias for the container.
+  - **ip=IPv4**: Specify a static ipv4 address for this container.
+  - **ip=IPv6**: Specify a static ipv6 address for this container.
+  - **mac=MAC**: Specify a static mac address address for this container.
+  - **interface_name**: Specify a name for the created network interface inside the container.
+- **\<network name or ID\>[:OPTIONS,...]**: Connect to a user-defined network, this is just the network name or ID from a network created by **[podman network create](podman-network-create.1.md)**. Using the network name implies the network mode bridge. It is possible to specify same options as described under bridge above. You can use **--network** multiple times to specify additional networks.
```

Comment:
- **\<network name or ID\>[:OPTIONS,...]**: Connect to a user-defined network; this is the network name or ID from a network created by **[podman network create](podman-network-create.1.md)**. Using the network name implies the bridge network mode. It is possible to specify the same options described under the bridge mode above. You can use the **--network** option multiple times to specify additional networks.